packages/babel-plugin-transform-remove-console/README.md

```diff
@@ -29,11 +29,19 @@ npm install babel-plugin-transform-remove-console
 **.babelrc**
 
 ```json
+// without options
 {
   "plugins": ["transform-remove-console"]
 }
 ```
 
+```json
+// with options
+{
+  "plugins": ["transform-remove-console", { "exclude": [ "error", "warn"] }]
```

Comment:
It should be `[ ["transform-remove-console", { ... } ] ]`. It's an array of array of two items (name & options).